http/proxy_query_service.go

```diff
@@ -9,37 +9,50 @@ import (
 	"net/http"
 
 	"github.com/influxdata/flux"
+	"github.com/influxdata/flux/iocounter"
+	icontext "github.com/influxdata/influxdb/context"
 	"github.com/influxdata/influxdb/query"
 	"github.com/julienschmidt/httprouter"
 	"github.com/prometheus/client_golang/prometheus"
 	"go.uber.org/zap"
 )
 
 const (
-	proxyQueryPath = "/api/v2/queryproxysvc"
+	proxyQueryPath    = "/api/v2/queryproxysvc"
```

Comment:
should we rename this to be `/queryproxysvc` since its not really part of the public API?

Reply:
Is there a convention in place for public vs. internal URLs?

Reply:
Not to my knowledge.